Terminal.Gui/ConsoleDrivers/CursesDriver/CursesDriver.cs

```diff
@@ -338,7 +342,13 @@ void ProcessInput ()
 
 			if (code == Curses.KEY_CODE_YES) {
 				if (wch == Curses.KeyResize) {
```

Comment:
I think this can be simplified to:

```cs
			while (code == Curses.KEY_CODE_YES && wch == Curses.KeyResize) {
				ProcessWinChange ();
				code = Curses.get_wch (out wch);
			}
			if (wch == 0) {
				return;
			}
```

Right?

Reply:
Done with https://github.com/gui-cs/Terminal.Gui/pull/2786/commits/6383afa998abf20d5b784df249f641d69494d91c.